src/styles/blocks/calendar.scss

```
@@ -0,0 +1,47 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: setWidth($blockSize,$daysOfWeeks,$marginBetweenBlocks);
+
+  &--start-day {
+    @include setFirstDayOfWeek;
+  }
+
+  &--month-length {
+    @include hideDaysOfMonth;
+  }
+
+  &__day {
+    position: relative;
+
+    @include setBlockSize;
+
+    margin-left: $marginBetweenBlocks;
+    margin-bottom: $marginBetweenBlocks;
+
+    background-color: #eee;
+    border: 1px solid black;
+    top: 0;
+    transition: color 0.5s, top 0.5s;
```

don't forget to create variables for transition duration & all colors 
```suggestion
    background-color: $c-color;
    border: 1px solid $c-black;
    top: 0;
    transition: color $transitionDuration, top $transitionDuration;
```